src/pages/certificates/tests/create-certificate.spec.js

```diff
@@ -41,6 +41,7 @@ describe('test certificates Emulation and Generation', () => {
     userEvent.click(getAllByRole('checkbox')[0]);
     fireEvent.change(input, { target: { value: 'john.dee@someemail.com' } });
     fireEvent.focusOut(input);
+    await new Promise((resolve) => setTimeout(resolve, 700));
```

Comment:
do we need this?
if you need to wait you can use waitFor()
https://testing-library.com/docs/dom-testing-library/api-async/

Reply:
He wouldn't use waitFor in our RTL version 
findBy query is better choise

Reply:
I deleted this part of test after i changed the code. This is not necessary here.